jina/peapods/remote.py

```diff
@@ -58,6 +60,12 @@ def loop_body(self):
         else:
             self.logger.error(f'fail to create {typename(self)} remotely')
 
+    def send_terminate_signal(self) -> None:
+        """Gracefully close this pea and release all resources """
+        if self.is_ready_event.is_set() and hasattr(self, 'ctrl_addr'):
+            send_ctrl_message(address=self.ctrl_addr, cmd='TERMINATE',
```

Comment:
isn't this command an enum?

Reply:
https://github.com/jina-ai/jina/blob/remote-log-endpoint/jina/peapods/pea.py#L407

Reply:
did it change recently?

Reply:
ah ok I guess is the new type redesign